src/style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.item {
+  color: white;
+  height: 300px;
+  width: calc(100%/3);
+  min-width: 300px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+
+}
+
+.item-1 {
+  background-color: rgb(255,0,0);
+
+}
+
+.item-2 {
+  background-color: rgb(204,0,0) ;
+
+}
+
+.item-3 {
+  background-color: rgb(153,0,0) ;
+  order: 1;
```

Comment:
You don't need to provide order property to those elements, which order should be the same as in HTML.